assets/store_descriptions/en-US/strings.xml

```diff
@@ -1,30 +1,30 @@
-    <supported>Supported Products</supported>
-    <bindings>Over 200 specific add-ons provide support for brands, devices, technologies and communication protocols. Examples are Z-Wave, Philips Hue, Amazon Echo, Chromecast and Sonos. Discover all available add-ons and supported devices and/or functions at: https://docs.openhab.org/addons/bindings.html</bindings>
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+    <string name="intro">openHAB - "empowering the smart home" - vendor and technology agnostic open source home automation</string>
+    <string name="whatis">openHAB is a Java-based open-source home automation platform that integrates and combines a vast range of different smart home systems and technologies into one single solution.</string>
+    <string name="rules">On top of the unified abstraction layer all connected devices are available for the overarching automation rule engines and different user interfaces.</string>
+    <string name="supported">Supported Products</string>
+    <string name="bindings">Over 200 specific add-ons provide support for brands, devices, technologies and communication protocols. Examples are Z-Wave, Philips Hue, Amazon Echo, Chromecast and Sonos. Discover all available add-ons and supported devices and/or functions at: https://docs.openhab.org/addons/bindings.html</string>
 
-    <home_automation_solutions>• Home Automation Solutions: Z-Wave, EnOcean, Netatmo, Homematic, Insteon, ...</home_automation_solutions>
-    <lighting>• Lighting: Philips Hue, Ikea Trådfri, LIFX, Lutron, Milight, ...</lighting>
-    <heating>• Heating: Max!, Nest, Vitotronic, Heatmiser, ...</heating>
-    <home_entertainment>• Home Entertainment: Samsung TV, LG TV, Sonos, Pioneer AVR, Squeezebox, Kodi, Plex, ...</home_entertainment>
-    <security>• Security: ZoneMinder, DSC, ...</security>
-    <open_protocols>• Open Protocols: HTTP, TCP/UDP, MQTT, Serial, ...</open_protocols>
-    <special_useCases>• Special UseCases: Minecraft, Tesla Car, Weather Services, ...</special_useCases>
-    <empty_point>• ...</empty_point>
+    <string name="home_automation_solutions">• Home Automation Solutions: Z-Wave, EnOcean, Netatmo, Homematic, Insteon, ...</string>
+    <string name="lighting">• Lighting: Philips Hue, Ikea Trådfri, LIFX, Lutron, Milight, ...</string>
+    <string name="heating">• Heating: Max!, Nest, Vitotronic, Heatmiser, ...</string>
+    <string name="home_entertainment">• Home Entertainment: Samsung TV, LG TV, Sonos, Pioneer AVR, Squeezebox, Kodi, Plex, ...</string>
+    <string name="security">• Security: ZoneMinder, DSC, ...</string>
+    <string name="open_protocols">• Open Protocols: HTTP, TCP/UDP, MQTT, Serial, ...</string>
+    <string name="special_useCases">• Special UseCases: Minecraft, Tesla Car, Weather Services, ...</string>
+    <string name="empty_point">• ...</string>
```

Comment:
I'd once again suggest to break those into their components. Imho the latter parts ("ZoneMinder, DSC, ...") are non-translatable and the item marker should imho not be part of the translation. There are just a few languages that are RTL and it shouldn't be a big task to add a list of them. While at it I'd suggest to indent the item marker with an enspace character (`&ensp;`).

Reply:
> are non-translatable

Are they? In non english familiar languages company and product names may be translated, e.g. the following image says "Overwatch Retribution" (Video game by blizzard) in Chinese:
http://s.newsweek.com/sites/www.newsweek.com/files/styles/embed-lg/public/2018/04/03/overwatch-chinese-venice-incident.png
Another example is the word firewall in windows. In German it's "Windows-Firewall", but in the Russian version it is literary translated.
I would put everything but the `<b>` tags in the translatable strings.

Reply:
Edge cases I'd say. The only ones I'd expect to be translatable are "Tesla Car, Weather Services".
It would be fine with me to leave the category and string as one string but I'd remove all formatting elements from the string (the item marker). Anyhow, as this project is your turf, I'll leave this decision with you.